modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -11,60 +11,43 @@
 .Default CNI network plugin feature comparison
 [cols="50%,25%,25%",options="header"]
 |===
-ifeval::["{context}" == "about-ovn-kubernetes"]
-|Feature|OVN-Kubernetes|OpenShift SDN
 
-|Egress IPs|Supported|Supported
-
-|Egress firewall ^[1]^|Supported|Supported
-
-|Egress router|Supported ^[2]^|Supported
-
-|Hybrid networking|Supported|Not supported
-
-|IPsec encryption for intra-cluster communication|Supported|Not supported
-
-|IPv6|Supported ^[3]^ ^[4]^|Not supported
-
-|Kubernetes network policy|Supported|Supported
-
-|Kubernetes network policy logs|Supported|Not supported
-
-|Hardware offloading|Supported|Not supported
-
-|Multicast|Supported|Supported
-endif::[]
-ifeval::["{context}" == "about-openshift-sdn"]
 |Feature|OpenShift SDN|OVN-Kubernetes
 
 |Egress IPs|Supported|Supported
 
-|Egress firewall ^[1]^|Supported|Supported
+|Egress firewall|Supported|Supported ^[1]^
 
 |Egress router|Supported|Supported ^[2]^
 
 |Hybrid networking|Not supported|Supported
 
-|IPsec encryption|Not supported|Supported
+|IPsec encryption for intra-cluster communication|Not supported|Supported
+
+|IPv4 single-stack|Supported|Supported
+
+|IPv6 single-stack|Not supported|Supported ^[3]^
 
-|IPv6|Not supported|Supported ^[3]^ ^[4]^
+|IPv4/IPv6 dual-stack|Not Supported|Supported ^[4]^
+
+|IPv6/IPv4 dual-stack|Not supported|Supported ^[4]^
 
 |Kubernetes network policy|Supported|Supported
 
 |Kubernetes network policy logs|Not supported|Supported
 
+|Hardware offloading|Not supported|Supported
+
 |Multicast|Supported|Supported
 
-|Hardware offloading|Not supported|Supported
-endif::[]
 |===
 [.small]
 --
 1. Egress firewall is also known as egress network policy in OpenShift SDN. This is not the same as network policy egress.
 
 2. Egress router for OVN-Kubernetes supports only redirect mode.
 
-3. IPv6 is supported only on bare metal, vSphere, {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+3. IPv6 single-stack networking on a bare-metal platform.
 
-4. IPv6 single stack is not supported on {ibm-power-name}, {ibm-z-name}, and Red Hat OpenStack clusters.
+4. IPv4/IPv6 or IPv6/IPv4 dual-stack networking on bare-metal, {vmw-full} (installer-provisioned infrastructure installations only), {ibm-power-name}, {ibm-z-name}, and {rh-openstack} platforms.
```

Comment:
Once you start the backport, note that OpenStack is supported on both dual--stack Ipv4 primary and IPv6 primary since 4.15. Before that, on 4.14 only IPv4 primary was available as Tech Preview